Podfile

```diff
@@ -3,7 +3,7 @@ platform :ios, '11.0'
 abstract_target 'fearlessAll' do
   use_frameworks!
 
-  pod 'FearlessUtils'
+  pod 'FearlessUtils', :git => 'https://github.com/soramitsu/fearless-utils-iOS.git', :commit => '03651d23cc470e05068b63095417ecb4170869b1'
```

Comment:
I think we have already updated the library in cocoapods and the latest version should be ok